src/machine/usb/adc/midi/midi.go

```diff
@@ -53,24 +54,40 @@ func newMidi() *midi {
 				Index:     usb.MIDI_ENDPOINT_IN,
 				IsIn:      true,
 				Type:      usb.ENDPOINT_TYPE_BULK,
-				TxHandler: m.Handler,
+				TxHandler: m.TxHandler,
 			},
 		},
 		[]usb.SetupConfig{},
 	)
 	return m
 }
 
+// SetHandler is now deprecated, please use SetRxHandler().
 func (m *midi) SetHandler(rxHandler func([]byte)) {
+	m.SetRxHandler(rxHandler)
+}
+
+// SetRxHandler sets the handler function for incoming MIDI messages.
+func (m *midi) SetRxHandler(rxHandler func([]byte)) {
 	m.rxHandler = rxHandler
 }
 
+// SetTxHandler sets the handler function for outgoing MIDI messages.
+func (m *midi) SetTxHandler(txHandler func()) {
```

Comment:
Maybe the TxHandler could receive the sent bytes? It would allow for users to inspect outgoing messages and react based on them, i.e: RGB LEDs color based on pitch?

Reply:
That is good idea, but I think I will leave it for a future implementation for now.